examples/workspaces_variables/README.md

```diff
@@ -0,0 +1,55 @@
+* Create a Terraform Cloud or Enterprise token to use with the `tfe` [provider](https://registry.terraform.io/providers/hashicorp/tfe/latest/docs). While other token types can be used, a [User Token](https://developer.hashicorp.com/terraform/cloud-docs/users-teams-organizations/users#api-tokens) is recommended in the beginning so that the provider has the same access as the user experimenting with this configuration.
+* Create a [Github token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) that has access to create and manage OAuth clients. A personal access token should be sufficient. This example uses Github, so the configuration will need to be modified to work with other VCS providers.
+
+Have the following information handy for the next steps:
+
+* TFC or TFE organization name
+* TFC or TFE token
+* Github organization name
+* Github repository names (`managed` and `manager`, but you could, and eventually probably will, rename them)
+* Github token
+
+### Setting up initial bootstrap infrastructure
+
+* Create two Github repositories named `manager` and `managed`.
+* Add the `manager` configuration to the `manager` repository.
+* Add the `managed` configuration to the `managed` repository.
+* Create a TFC/TFE workspace named `manager` in the organization that was created or joined.
+  - If there is no existing connection for Github configured, select to [add a connection](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/settings/vcs).
+  - With Github connected, choose the `manager` repository from the list.
+  - Use the default options for the remaining settings to complete the workspace setup.
+
+### Manager workspace variables
+
+The `manager` workspace requires certain Terraform variables to be set on the [Variables page in the UI](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/variables#managing-variables-in-the-ui). All variables should be "Terraform Variables". There are no "Environment Variables" required.
+
+* `tf_hostname` - The Terraform Cloud or Enterprise hostname. Defaults to app.terraform.io
+* `tf_api_token` - The TFC/TFE user token that was generated. Mark this as sensitive.
+* `tf_organization` - The TFC/TFE organization name where the managed workspaces should exist. This will most commonly be the organization that was created or joined, where the `manager` workspace also exists. The token supplied to perform the operations should have sufficient access to the organization.
+* `vcs_repo_identifier` - A reference to the `managed` repository in the format <github-organization>/<repository>. The format of the VCS repo identifier might differ depending on the VCS provider, see https://registry.terraform.io/providers/hashicorp/tfe/latest/docs/resources/workspace
```

Comment:
I noticed the preview doesn't display the values in carrots. I would recommend adding backticks `<github-organization>/<repository>`